website/content/commands/intention/check.mdx

```diff
@@ -7,6 +7,8 @@ page_title: 'Commands: Intention Check'
 
 Command: `consul intention check`
 
+Corresponding HTTP API Endpoint: [\[GET\] /v1/connect/intentions/check](https://www.consul.io/api-docs/connect/intentions#check-intention-result)
+
```

Comment:
My spreadsheet was actually missing the ACL required for this (just added it). It should be `intentions:read`. Can we add that here?

It also comes with a footnote on the API endpoint page.

![image](https://user-images.githubusercontent.com/85913323/148857828-3f3d9d6b-4e05-46c6-b837-3fa7b41e2f59.png)

Reply:
Added.  Also updated all intentions ACLs to include the footnote (as the API docs do).